editor/uiclasses.js

```diff
@@ -173,6 +173,13 @@ class Node {
     getFormula() {
         throw new Error("This method must be overridden in a subclass");
     }
+
+    /**
+     * @returns The asymptotes of the node block, considering all sub-nodes.
+     */
+    getAsymptotes(xMin, xMax) {
+        throw new Error("This method must be overridden in a subclass");
```

Comment:
yo wait so thats how u use the try function? i thought it just made all the errors go away lol
this is so good

Reply:
no i don't use the try function here